llms/openai/multicontent_test.go

```diff
@@ -162,6 +162,20 @@ func TestFunctionCall(t *testing.T) {
 	assert.NotNil(t, c1.FuncCall)
 }
 
+func TestTranscription(t *testing.T) {
+	t.Parallel()
+	llm := newTestClient(t, WithModel("whisper-1"))
+
+	audioFilePath := "./sample.mp3"
+	_, err := os.Stat(audioFilePath)
+	require.NoError(t, err)
+
+	rsp, err := llm.TranscribeAudio(context.Background(), audioFilePath)
```

Comment:
Does it make sense to think of "transcribe audio" in the context of LLMs? AFAIU Whisper is a distinct model from LLMs like the GPT family.

Is this intended as a one-off method only for openai, or as some general audio transcription interface?

Reply:
only for openai, but I think it's interesting to include it in the general context, because there are other models that do this, but at the moment it's only for openai

Reply:
I haven't tried it yet, but this seems interesting as a locally running alternative: https://github.com/JigsawStack/insanely-fast-whisper-api 

It would be cool if we could support something like that, so you can combine it with a Ollama to build some local-only tools.

(So far I've been using https://github.com/Purfview/whisper-standalone-win locally, which is a single-binary wrapper around https://github.com/SYSTRAN/faster-whisper)

Reply:
We can implement it in other LLMS, the problem is that so far I have only found the standalone version for Windows, but I am looking for other alternatives